02_what_why_intro.Rmd

```diff
@@ -1,20 +1,107 @@
-# What is Data Visualization?
+# What is data visualization?
 
 ```{r, include = FALSE}
 ottrpal::set_knitr_image_path()
 ```
 
 ## Learning Objectives
 
-This chapter will cover:
-
-- {LO1}
-- {LO2}
+```{r fig.align = 'center', out.width = "100%", echo = FALSE, fig.alt = "Chapter specific LOs go here"}
+ottrpal::include_slide("https://docs.google.com/presentation/d/1fu-KfdN2ldOXB49o9zdpurFbVvl1bOh8cfQ3vu0szk4/edit?slide=id.g3015fd53a67_4_176#slide=id.g3015fd53a67_4_176")
+```
 
 ## What is Data Visualization?
 
-### Exploratory Data Analysis (EDA)
+<div class = "dictionary">
+**Data visualization** is the process (verb) or end product (noun) of making graphical representations such as plots or other figures to show different representations of information, to both explore or communicate aspects such as trends, patterns, relationships, or steps in a process.
+</div>
+
+A data visualization may show every data point within a dataset (raw data), or it may show transformations, subsets, statistical summaries, confidence intervals, or estimates of relationships between variables within the dataset. It may represent data using graphics such as points, lines, bars, area/size, color, shape, or some combination of these. Visualizations can also be used to provide an overview of a process, rather than displaying specific data. By whatever means the information is displayed, a complete or polished data visualization will also include design elements beyond graphics: informative text such as titles, axis labels, annotations or direct labeling, and captions. These are the building blocks of graphical representation of data. They may be combined in specific ways to form standard plot formats such as scatter plots, pie charts, bar plots, histograms, and more (with even more specialized formats used in certain industries and research fields - e.g., Sankey Diagrams, Forest Plots, Kaplan-Meier Curves, etc.).
+
+```{r fig.align = 'center', out.width = "100%", echo = FALSE, fig.alt = ""}
```

Comment:
```suggestion
```{r fig.align = 'center', out.width = "100%", echo = FALSE, fig.alt = "Data visualizations summarize information, representing it in a less complex manner"}
```